src/helpers/cosmiconfig.ts

```diff
@@ -40,9 +40,24 @@ export function createCosmiConfigSync(moduleName: string, legacy: boolean) {
 }
 
 const loadTypeScript: Loader = (...args) => {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { TypeScriptLoader } = require('cosmiconfig-typescript-loader');
+    return TypeScriptLoader({ transpileOnly: true })(...args);
```

Comment:
@beerose do we still need `cosmiconfig-typescript-loader`?

Reply:
Yeah, it looks like we don't! Just pushed the changes